test/Serilog.Tests/Support/StringSink.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Globalization;
+using System.IO;
+using Serilog.Core;
+using Serilog.Events;
+using Serilog.Formatting;
+using Serilog.Formatting.Display;
+
+namespace Serilog.Tests.Support
+{
+    public class StringSink : ILogEventSink
+    {
+        readonly StringWriter _sw = new StringWriter();
+        readonly ITextFormatter _formatter;
+
+        const string DefaultOutputTemplate = "{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} [{Level}] {Message}{NewLine}{Exception}";
```

Comment:
Probably depends on the test, seems like a good change but trying to keep the port 1:1 for now.